### PR TITLE
perf(benchmarks): Add two markdown benchmarks

### PR DESCRIPTION
**Pull request by pvdz**
This adds two new MD based benchmarks and renames the old one.

The old one was using tables (useful but heavily skewing perf results to processing tables in the plugin) and generating the files at benchmark time, rather than before it. So I renamed that one to `markdown_table`.

The two new sites are basically the same except one uses `slug` to index pages and the other one `id`. We optimize internally for `id` and you can see that when comparing the biuld time of these two sites. The difference grows and is clearly noticable at 10k+ pages.

See READMEs for details.

The two new sites will be data points for the benchmarking service we're setting up.

### CODE REVIEW

**Comment by pvdz** [comment]
cc @DSchau @nicholascapo

File: benchmarks/markdown_id/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "devDependencies": {
+    "faker": "^4.1.0",
+    "prettier": "^1.19.1"
+  },
+  "keywords": [
+    "gatsby"
+  ],
+  "license": "MIT",
+  "main": "n/a",
+  "scripts": {
```
Comment (nicholascapo) [inline benchmarks/markdown_id/package.json]: I would prefer a `postinstall` script that just runs `NUM_PAGES=${NUM_PAGES:-2000} node md.generate.js`

File: benchmarks/markdown_slug/package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  "devDependencies": {
+    "faker": "^4.1.0",
+    "prettier": "^1.19.1"
+  },
+  "keywords": [
+    "gatsby"
+  ],
+  "license": "MIT",
+  "main": "n/a",
+  "scripts": {
```
Comment (nicholascapo) [inline benchmarks/markdown_slug/package.json]: I prefer a `postinstall` script here as well

File: benchmarks/markdown_id/package.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
+    "bench": "rm -r markdown-pages; NUM_PAGES=${NUM_PAGES:-2000} node md.generate.js; gatsby clean; node --max_old_space_size=2000 node_modules/.bin/gatsby build",
```
Comment (nicholascapo) [inline benchmarks/markdown_id/package.json]: Sorry I should have been more specific: `generate` seems like a part of `postinstall`, but I think `build` should be a separate script, and not automatic as part of `yarn install`.

Comment (pvdz) [reply benchmarks/markdown_id/package.json]: Got it!

**Comment by pvdz** [comment]
Merging (This is not affecting any existing code, nor tests)

**Pull request merged**